### ISSUE TITLE
badge: selecting a button with a badge show focus highlighting includiong the badge  that looks unconventional

### ISSUE THREAD

**Issue by hramakumar** [eBay/ebayui-core#489]
# Bug Report

## Description
go to https://go/ebayui/test
in the badge section click on the one with the heading badge-button-icon
look at the focussed outline once its clicked
## Screenshots
https://www.useloom.com/share/bbde58ad2f5e45e69decfd27defaf370

**Comment by seangates** [comment]
@hramakumar That actually looks good and correct to me.

**Comment by hramakumar** [comment]
I believe it should either highlight the button alone and not the badge or it should have a bigger box that includes the button and the badge. The current shape looks irregular
![image](https://user-images.githubusercontent.com/13157115/49554737-344b0900-f8b2-11e8-931b-b7eec4131c3f.png)

**Comment by seangates** [comment]
@ianmcburnie Your thoughts?

**Comment by ianmcburnie** [comment]
I think it would look better just around the button if we can do that, but I don't think it's urgent. Moving to backlog as a good first issue.

**Comment by ianmcburnie** [comment]
Yeah, I don't think moving the badge element outside of the button element is an option at this point (or maybe ever), so if there's no way to fix this with the current DOM, we'll just have to live with it.

**Comment by ianmcburnie** [comment]
Safari doesn't have this issue btw:

<img width="52" alt="screen shot 2018-12-14 at 6 02 15 pm" src="https://user-images.githubusercontent.com/38065/50037829-7cf97500-ffca-11e8-8650-264839fbe494.png">
